src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -535,7 +538,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_gamma)
 
 // gammaln
 MXNET_OPERATOR_REGISTER_UNARY(gammaln)
-.MXNET_DESCRIBE("Log of the absolute value of the gamma function, element-wise")
+.MXNET_DESCRIBE("Returns element-wise Log of the absolute value of the gamma function of the input.")
```

Comment:
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.gammaln.html

The leading Returns seems superfluous.